trieste/bayesian_optimizer.py

```diff
@@ -492,39 +492,45 @@ def optimize(
 
                 summary_writer = get_tensorboard_writer()
                 if summary_writer:
-                    with summary_writer.as_default():
+                    with summary_writer.as_default(step=step):
                         for tag in datasets:
                             with tf.name_scope(f"{tag}.model"):
                                 models[tag].log()
-                            tf.summary.scalar(
-                                f"{tag}.observation.best_overall",
-                                np.min(datasets[tag].observations),
-                                step=step,
+                            tf.summary.histogram(
+                                f"{tag}.observation/new_observations",
+                                tagged_output[tag].observations,
                             )
                             tf.summary.scalar(
-                                f"{tag}.observation.best_new",
+                                f"{tag}.observation/best_in_batch",
                                 np.min(tagged_output[tag].observations),
```

Comment:
It doesnt really make sense to define the best like this when observations are noisy.  Perhaps its better to name this "best_obeservation_in_batch" just to be clear